app/src/main/java/eu/tutorials/futureweatherforecast/activity/MainActivity.kt

```diff
@@ -180,6 +141,61 @@ class MainActivity : AppCompatActivity() {
 
     }
 
+    override fun onCreateOptionsMenu(menu: Menu?): Boolean {
+        menuInflater.inflate(R.menu.custom_actionbar,menu)
+        return true
+    }
+
+    @RequiresApi(Build.VERSION_CODES.N)
+    override fun onOptionsItemSelected(item: MenuItem): Boolean {
+        return when(item.itemId) {
+            R.id.defaultLocation -> {
+                    getLocationWeatherDetails()
+                Toast.makeText(this, "Current Location", Toast.LENGTH_SHORT).show()
+                return true
+            }
+//            R.id.Theme -> {
+//
+//                return true
+//            }
+            R.id.Lightmode -> {
+                bindind.MainBG.setBackgroundResource(R.drawable.bgmorning)
+                bindind.tempText.setTextColor(Color.BLACK)
+                return true
+            }
+            R.id.Darkmode -> {
+                bindind.MainBG.setBackgroundResource(R.drawable.bgcloudynight)
+                bindind.tempText.setTextColor(Color.WHITE)
+                return true
+            }
+//            R.id.Temperature -> {
+//
+//                return true
+//            }
+            R.id.Celcius -> {
+                Constants.TemperatureScale = "C"
```

Comment:
you can create `MainViewModel` here and update it using livedata

Reply:
its not good idea, that you update the constant variable here as we can use livedata to update the value